.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@ repos:
           )$
       - id: check-merge-conflict
   - repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 22.3.0
     hooks:
       - id: black
         language_version: python3
```

Comment:
Hi, maybe we also need to pin `click` to `8.0.4` in `black`
```
  - repo: https://github.com/psf/black
    rev: 22.3.0
    hooks:
      - id: black
        additional_dependencies: ["click==8.0.4"]
        language_version: python3
```

Reply:
It looks like we need to account for some formatting changes that come with the version update.